packages/client/wallets/smart-wallet/src/blockchain/chains.ts

```diff
@@ -29,13 +43,21 @@ export const zerodevProjects: Record<SmartWalletChain, string> = {
     "polygon-amoy": "3deef404-ca06-4a5d-9a58-907c99e7ef00",
     "base-sepolia": "5a127978-6473-4784-9dfb-f74395b220a6",
     base: "e8b3020f-4dde-4176-8a7d-be8102527a5c",
+    "optimism-sepolia": "f8dd488e-eaed-467d-a5de-0184c160f3b1",
```

Comment:
I'm curious that we weren't supporting this. I remember testing these chains some months ago, but probably something changed.

Reply:
The main repo supports them out of the box as the default contract version configuration works for it and we have ZeroDev projects configured for it, it was just the SDK was never adapted so these are supported chains. Here we create SDK-specific ZeroDev projects and add these chains to the SDK

Reply:
@MatiNavCrossmint I think we dropped support for them at one point to just say polygon & base work